checkmate/async/tasks.py

```diff
@@ -0,0 +1,20 @@
+"""Async tasks."""
+
+from celery.utils.log import get_task_logger
+
+from checkmate.async.celery import app
+from checkmate.checker.url.blocklist import Blocklist
+
+LOG = get_task_logger(__name__)
+
+
+@app.task
```

Comment:
```suggestion
@celery.task(acks_late=False)
```

As mentioned above you don't want Celery to pass the task to another / the same worker again if the worker crashes and fails to acknowledge the task. You just want it to be retried when next scheduled. So you want the task to be acknowledged early

Reply:
I think I prefer your suggestion above. If all of our tasks are going to be like this, we can just set the default in the celery config.

This really is `app.task`. We have made the decision to have a file called `celery` with a variable called `celery` in it in H and to also import and re-export things from the package `celery`, which makes this all quite confusing. This follows the pattern in the celery docs: https://docs.celeryproject.org/en/stable/getting-started/first-steps-with-celery.html#application